### PR TITLE
feat(i18n): Update French translations

### PR DESCRIPTION
**Pull request by KaKi87**
- Removed 3 unused keys
- Added 16 missing keys
- Edited 4 "cacher" -> "masquer" keys
- Fixed "répondre à" -> "réponse à" key

### CODE REVIEW

File: locales/fr-FR.json [hunk 1]
```diff
@@ -49,8 +49,8 @@
     "confirm": "Confirmer",
     "edit": "Éditer",
     "enter_app": "Entrer dans l'application",
-    "favourite": "Ajouter aux messages aimés",
-    "favourited": "Ajouté aux messages aimés",
+    "favourite": "J'aime",
+    "favourited": "J'aime",
```
Comment (edimitchel) [inline locales/fr-FR.json:53]: 👍🏻

**Review by edimitchel — CHANGES_REQUESTED** [review]

File: locales/fr-FR.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "blocked_users": "Utilisateur·ice·s bloqué·e·s",
     "blocking": "Bloqué·e",
     "bot": "Automatisé",
-    "favourites": "Favoris",
+    "favourites": "Aimés",
```
[the remark this answers is not in the document]
Comment (KaKi87) [reply locales/fr-FR.json:16]: We ? I wasn't invited to a vote.

[the remark this answers is not in the document]
Comment (patak-dev) [reply locales/fr-FR.json:16]: @KaKi87 we don't use voting to decide about features in our projects. But a general evolving consensus and we trust in the decision-making of close contributors to the project. These decisions aren't final and may continue to change. I advise not getting attached to these decisions (there are tons of them every week) if you would like to properly collaborate with others in the project.

[the remark this answers is not in the document]
Comment (edimitchel) [reply locales/fr-FR.json:16]: Yes we did: https://github.com/elk-zone/elk/pull/1133#discussion_r1070430681

**Pull request closed without merging**